packages/core/src/types.ts

```diff
@@ -916,4 +920,9 @@ export interface GenerateOptions<T extends boolean> {
    * If return extended "matched" with payload and count
    */
   extendedInfo?: T
+
+  /**
+   * show origin theme color
+   */
+  showOriginThemeColor?: boolean
```

Comment:
We probably don't need a new option for that. We could always do that when `envMode` is `dev`